src/ai/backend/gateway/stream.py

```diff
@@ -388,6 +390,11 @@
 
     try:
         opts: Mapping[str, Any] = {}
+        if params['arguments'] != None:
```

Comment:
Use `is not` instead of `!=`.
See https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none
TL;DR: PEP-8 recommends `is` operator for `None` comparison and it is much faster.